### DISCUSSION TITLE
Text Effects

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [dlemstra/Magick.NET#1100] [Help]
Hi! I bought new font.
And it was beautiful at picture> but a cant make this text effects. 
Please help to understand. 
![Screenshot_1](https://user-images.githubusercontent.com/67605008/148257418-f6d46ed4-3c00-4dcb-ab29-b72be06e5f33.png)

Link to Dropbox to font file
https://www.dropbox.com/s/mrfvbv2shiemrdj/Im%20a%20little%20chicken.ttf?dl=0

**Comment by dlemstra** [comment]
I don't understand what you are asking me?

**Comment by jaka1-newbie** [comment]
I want make text with effects as at image

**Reply by dlemstra** [reply]
What do you mean by text effects? I really don't understand what you are trying to ask me.

**Comment by jaka1-newbie** [comment]
![Screenshot_2](https://user-images.githubusercontent.com/67605008/151366388-d4140c63-89d6-4215-bede-c9cc0438569b.png)

**Comment by jaka1-newbie** [comment]
```
var settings = new MagickReadSettings
{
    BackgroundColor = MagickColors.None, // -background none
    FillColor = new MagickColor("#8AAC10"), 
    Font = @"C:\Windows\Fonts\Im a little chicken.ttf", 
    Width = 1536,
    Height = 864, // -size 1536x864
	StrokeColor = new MagickColor("#68840B"),
	StrokeWidth=3,
    TextGravity = Gravity.Center, // -gravity center
};
using (var caption = new MagickImage("caption:Chiken Friends", settings))
{
    using (var overlay = caption.Clone()) // +clone
    {
        overlay.BackgroundColor = MagickColors.Black;
        overlay.Shadow(5, 5, 8, new Percentage(50));
        overlay.Composite(caption, CompositeOperator.Over);
		
 using (var background = new MagickImage("xc:#FFF4C2", settings))
        {
            background.Composite(overlay, Gravity.Center, CompositeOperator.Over);
            background.Write("a.jpg");
        }
    }
}
```
Solved.
Wanted this.


One more question: Can Magick.Net find darker color near another?
For example:  This (#8AAC10) for this (#68840B)

**Comment by jaka1-newbie** [comment]
Get code of color more darker then original...
